lms/templates/shoppingcart/receipt.html

```diff
@@ -183,97 +183,97 @@ <h3 class="text-center">${_("Please send each professional one of these unique r
             <div class="row-inside">
               <p>
                 <b>${_('Card Type')}:</b>
-                <label>
+                <span>
                   % if order.bill_to_cardtype:
-                    ${order.bill_to_cardtype}
+                    ${_('{bill_to_cardtype}'.format(bill_to_cardtype=order.bill_to_cardtype))}
```

Comment:
I lead @dino-cikatic to believe that should be translated but upon closer examination of the requirements I can see that translation is not a requirement there.